documentation/overview_and_index.md

```diff
@@ -0,0 +1,54 @@
+## Problem Statement
+[Human Rights First (HRF)](https://www.humanrightsfirst.org/)   is an independent advocacy and 
+action organization that challenges America to live up 
+to its ideals. HRF is seeking a data search and 
+visualization tool to provide the public with a 
+user-friendly view into their large data set of asylum 
+case decisions. Similar to [the TRAC database](https://trac.syr.edu/phptools/immigration/asylum/)—
```

Comment:
Miswording -- change to "The concept is similar to TRAC, a popular existing tool, but with a far more robust ability to select and filter data." Otherwise, it sounds like you're saying it will be like TRAC but with more limitations.

Reply:
I like this edit a lot.